hbt/tasks/sync.py

```diff
@@ -183,7 +183,11 @@ def requires(self):
 
     @workflow_condition.output
     def output(self):
-        return self.target(f"sync_{self.dataset_inst.name}_{self.branch}.csv")
+        return {
+            "normal": self.target(f"sync_{self.dataset_inst.name}_{self.branch}.csv"),
+            "hhbtag": self.target(f"sync_{self.dataset_inst.name}_{self.branch}_hhbtag.csv"),
+            "ressonant": self.target(f"sync_{self.dataset_inst.name}_{self.branch}_ressonant.csv"),
```

Comment:
```suggestion
            "resonant": self.target(f"sync_{self.dataset_inst.name}_{self.branch}_resonant.csv"),
```
Again, typo, but I guess that one is propagated to the sync tools since the typo is also in the name of the file?